### PR TITLE
Maap documentation tutorials fix for #127 and #128

### PR DESCRIPTION
**Pull request by sumantjha**
Maap documentation tutorials fix for #128 and #127 

#128 GEDI L2B Notebook Syntax error
#127 TOC incorrect levels.

### CODE REVIEW

File: docs/source/tutorials/HLS/HLS_search.ipynb [hunk 1]
```diff
@@ -83,7 +83,7 @@
    "id": "25576924",
    "metadata": {},
    "source": [
-    "## Creating an AOI\n",
+    "Creating an AOI: \n",
```
Comment (wildintellect) [inline docs/source/tutorials/HLS/HLS_search.ipynb:86]: @sumantjha why did you remove the markdown heading?

Comment (sumantjha) [reply docs/source/tutorials/HLS/HLS_search.ipynb:86]: That was causing the TOC headings in the html. I double checked by running Sphinx on local. Without the markdown heading, the TOC looks as it should.

Comment (sumantjha) [reply docs/source/tutorials/HLS/HLS_search.ipynb]: > We want those headings in the TOC, example: image

Done!

**Review by wildintellect — APPROVED** [review]
